res/values/strings.xml

```diff
@@ -594,6 +594,12 @@
     <string name="systemmsg_ephemeral_timer_week">Disappearing messages timer set to 1 week.</string>
     <string name="systemmsg_ephemeral_timer_four_weeks">Disappearing messages timer set to 4 weeks.</string>
 
+    <!-- Translators: Protection disabled/enabled messages will be displayed together with a big "shield" and may be suffixed by systemmsg_action_by_user or systemmsg_action_by_me -->
+    <string name="systemmsg_chat_protection_disabled">Chat protection disabled.</string>
```

Comment:
```suggestion
    <string name="systemmsg_chat_protection_disabled">Chat protection off.</string>
```

Reply:
that would not fit when the suffix "by ..." is added. also imho, "enabled" implies some action taken before. this is not true for "on"

Reply:
Well, as anything other than a manifestation of itself, (which is weird). "Now" added.

Reply:
what's wrong with "Chat protection disabled"?

Reply:
The distinction between "off" and "disabled" is unimportant(?) so the primary result is more interesting.
Enabled/disabled is mechanical and off-putting. I thought the strings were consistently "on/off", but that got shut down https://github.com/deltachat/deltachat-android/pull/1122/files
Instead it is, both. That is however the type of consistency you will find.

Reply:
but the first message says "Chat protection enabled/disabled by XXXX" and so it's reasonable to re-use the same term. 
Bjoern: as noted somehwere else, i think it should be "chat protection enabled by last message" but maybe i am misinterpreting, only briefly in today.

Reply:
@hpk42 yip, you're right. the "shown in this chat" is a bit misleading. messages that were in the chat before do not have the protection.

maybe say. 
"Chat protection enabled [by me|username]. Subsequent messages are protected against compromised servers."
or  
"Chat protection enabled [by me|username]. Subsequent messages are protected against compromised servers until protection is disabled again."

to the context: the initial post is about two system-messages, one for enabling, one for disabling. we can also leave out the explaining text completely, however, i think it is good to describe what "Chat protection" means here, similar as what whatsapp is doing - even if not everyone understands what that means.

i'd add a screenshot here, however, this requires the shield first, which is not ready yet (and also not required for the next release where we do not have the on/off switch)

Reply:
Doesn't stating the same thing using the same words take away an opportunity to understand what is going on?